compose/local/postgres/postgres.conf

```diff
@@ -0,0 +1,750 @@
+# -----------------------------
```

Comment:
This config was not here before so I presume it there was a default that was part of the image that we were using before.

Why did we need to add this config, and are there any parameters in this config are now set differently than the default that was set before?

Reply:
We were using the production postgres inside local.yml.
Which is weird, for me.